### ISSUE TITLE
Server Dead... again

### ISSUE THREAD

**Issue by azulskyknight** [Y0ungSandwich/Mech-Magic-1.12.2#84]
3.1.5 from the start, running 6gb of RAM on the server.  It ran flawlessly without any issue for the last 10 days, it started as a resolution of issue #83.

The server is a Windows 10 Pro VM, running on HyperV 2016, on a Dell Poweredge T330 server.  

The Windows 10 VM rebooted last night for patches, and this morning I went to restart the server and a crash very similar to what took out my last server happened.  It simply won't start, complaining about an inconsistent world.

System is using AdoptOpen JRE 8.0.222.10 x64

So I moved the world folder out the of the way to generate a fresh world on the same seed, and with the idea there is something up with Immersive Petroleum's Distillation Tower.  I started testing.

Load into world, locate an oil source with Immersive Engineering's core drill, plop down a pump jack and a tank to store oil, power it with a creative battery.  Log out, stop server with stop command, restart server.  Server OK

Plop down Distillation Tower, connect it to the oil tank, flip the switch and add power, now it's making stuff.  Save and exit again. Once again stop the server and restart it.  Server OK

The single fluid output of the Distillation Tower requires the ability to filter three different liquids.  For some reason, this process doesn't work correctly with immersive fluid pipes connected to the tower, or with an immersive fluid router directly attached to the tower.  So instead I use a Thermal Expansion pipe to connect to the fluid router, and immersive pipes from there.  Connect the tanks to the appropriate outputs on the fluid router.  Once again stop the server, and restart it.  And server is OK...

So now I'm at a complete loss, this is the second time the server has died in two weeks with a corrupted map, and I'm utterly unclear as to what could be causing it.  In both cases I had just brought the Petroleum Tower online.  I assumed it was something with the fluid routing, but this test proves that assertion false.

Latest Log is here: https://pastebin.com/hNjfbSTq
Crash Report is here: https://pastebin.com/x5BLfEFS
World files are here: https://drive.google.com/file/d/12PqETNB6MYzxt_PJLWm1cy_Af1dHFi9q/view?usp=sharing
Seed: -1744572176

Mods I know people were using in their builds:
Immersive Engineering
Immersive Tech
Immersive Petroleum
Tinkers' Construct
Botania
Railcraft
Embers Rekindled
Blood Magic: Alchemical Wizardry

As this was the rebuild world there was a ton of creative work going on trying to get back the lost time from the last one.  As a result, all the big builds were right around the origin of the world.  Which means it's pretty easy to find everything.  MCEdit loads the world just fine as well to see where all the building was going on.

**Comment by Y0ungSandwich** [comment]
I'd roll back from a backup file or delete the region in question. Or use a sponge plugin like nucleus to Blockzap the coordinates of the corrupted entity. 

have you tried removing erroring entities with the forge config option?

`# Set this to true to remove any Entity that throws an error in its update method instead of closing the server and reporting a crash log. BE WARNED THIS COULD SCREW UP EVERYTHING USE SPARINGLY WE ARE NOT RESPONSIBLE FOR DAMAGES.
    B:removeErroringEntities=false`

**Comment by azulskyknight** [comment]
I forgot to mention, none of my backups work.  All of them behave the same way.  The only way to get the server to start was to use a blank world.  The last backup I have was shortly after the Petroleum Tower was added to the map, which is why I suspected it to begin with.

Both  B:removeErroringEntities=true, and  B:removeErroringTileEntities=true

Server still dies, and if there was anything in the log that gave me a solid idea of what Entity was doing this, I'd go happily nuke it.  

But all I have to go on are: Description: Exception in server tick loop
Some vague EnderIO warnings that are there even on a fresh map
and four of these before it dies. [Server thread/WARN] [net.minecraft.tileentity.TileEntity]: Skipping BlockEntity with id 

The server halts right when the mods are all loaded, and it'd ready to accept players.  If there are coordinates to a problem somewhere, either they aren't here or I don't know where to find them.
